pkg/core/object/fmt.go

```diff
@@ -147,28 +145,11 @@ func (v *FormatValidator) validateSignatureKey(obj *object.Object) error {
 		return fmt.Errorf("decode public key: %w", err)
 	}
 
-	token := obj.SessionToken()
-
-	if token == nil || !token.AssertAuthKey(&key) {
-		return v.checkOwnerKey(*obj.OwnerID(), key)
```

Comment:
This raises two questions:
 * when objects in a public container are replicated, we routinely have signatures from different keys (not container owner key), how this is controlled wrt EACL? One object can be created when this is allowed to happen (EACL rule allowing PUT for a key exists), then rules might change. We want to replicate the first object, but if a malicious node creates an object signed by some random keys and tries to replicate it this shouldn't be possible. Granted, this is not exactly a session token problem, but it's related.
 * won't this change break `token == nil` case? random keys signing objects are not good, maybe we have a private container.

Reply:
One big pile of problems is that NeoFS doesn't distinguish object creation and regular transmission like replication. Such a division of operations would clearly improve access management and prevent potential vulnerabilities in the form of malicious data generation by storage nodes.

`token == nil` condition removal doesn't break IAM since it is contolled on the other layer. Technically, object owner may be any according to the protocol and specification.